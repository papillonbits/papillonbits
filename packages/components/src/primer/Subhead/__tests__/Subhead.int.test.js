import { plain, spacious } from './Subhead.int.story'

describe('<Subhead />', () => {
  describe('Render', () => {
    test('must match plain()', () => {
      expect(global.renderToJSON(plain())).toMatchSnapshot()
    })

    test('must match spacious()', () => {
      expect(global.renderToJSON(spacious())).toMatchSnapshot()
    })
  })
})
