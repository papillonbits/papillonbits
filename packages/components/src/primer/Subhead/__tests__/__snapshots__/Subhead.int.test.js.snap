// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Subhead /> Render must match plain() 1`] = `
<div>
  <div
    class="Subhead"
  >
    <div
      class="Subhead-heading"
    >
      Plain subhead
    </div>
  </div>
</div>
`;

exports[`<Subhead /> Render must match spacious() 1`] = `
<div>
  <div
    class="Subhead Subhead--spacious"
  >
    <div
      class="Subhead-heading"
    >
      Spacious subhead
    </div>
  </div>
</div>
`;
